### PR TITLE
Implemented: functionality to save 'auto cancel days' (#2cxr1cx)

### PR DESCRIPTION
**Pull request by rathoreprashant**
### Related Issues

Closes #

### Short Description and Why It's Useful

Make a service (updateProductStore) and add a save button to the UI to save changes when the user updates auto cancel days settings.

### Screenshots of Visual Changes before/after (If There Are Any)

https://user-images.githubusercontent.com/33576503/182781351-8088bb61-9800-418f-aaf7-f73840c8b22f.mp4

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

**Review by dt2patel — APPROVED** [review]
Ui looks good

File: src/views/Orders.vue [hunk 1]
```diff
@@ -82,7 +82,10 @@
             </ion-card-header>
             <ion-item>
               <ion-label class="ion-text-wrap">{{ $t("Days") }}</ion-label>
-              <ion-input :placeholder="$t('before auto cancelation')" />
+              <ion-input :placeholder="$t('before auto cancelation')" v-model.number="autoCancelDays" type="number" />
```
Comment (adityasharma7) [inline src/views/Orders.vue:85]: I think we should also repopulate the already set value

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Orders.vue (hunk 1 above)
Comment (rathoreprashant) [reply src/views/Orders.vue:85]: updated sir

File: src/views/Orders.vue [hunk 2]
```diff
@@ -424,6 +446,17 @@ export default defineComponent({
       }
     });
     this.store.dispatch('webhook/fetchWebhooks')
+
+    const payload = {
+      "inputFields": {
+        'productStoreId': this.store.state.user.currentEComStore.productStoreId,
+      } as any,
```
Comment (ymaheshwari1) [inline src/views/Orders.vue:453]: I think we don't need `as any` here, please check and update.

File: src/views/Orders.vue [hunk 3]
```diff
@@ -424,6 +446,17 @@ export default defineComponent({
...
+      "fieldList": [ 'daysToCancelNonPay' ],
+      "entityName": "ProductStore",
+      "noConditionFind": "Y"
+    }
+    const resp = await JobService.getAutoCancelDays(payload);
```
Comment (ymaheshwari1) [inline src/views/Orders.vue:458]: Handle error cases

File: src/views/Orders.vue [hunk 4]
```diff
@@ -274,6 +279,23 @@ export default defineComponent({
         await this.store.dispatch('webhook/unsubscribeWebhook', { webhookId: webhook?.id, shopifyConfigId: this.shopifyConfigId })
       }
     },
+    async updateAutoCancelDays(autoCancelDays: number){
+      const payload = {
+        'productStoreId': this.store.state.user.currentEComStore.productStoreId,
```
Comment (ymaheshwari1) [inline src/views/Orders.vue:284]: Use getter to get current ecom store

File: src/views/Orders.vue [hunk 5]
```diff
@@ -424,6 +446,17 @@ export default defineComponent({
       }
     });
     this.store.dispatch('webhook/fetchWebhooks')
+
+    const payload = {
+      "inputFields": {
+        'productStoreId': this.store.state.user.currentEComStore.productStoreId,
```
Comment (ymaheshwari1) [inline src/views/Orders.vue:452]: Use getter to get current ecom store

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Orders.vue (hunk 2 above)
Comment (rathoreprashant) [reply src/views/Orders.vue]: Improved sir

File: src/views/Orders.vue (hunk 3 above)
Comment (rathoreprashant) [reply src/views/Orders.vue]: handled sir

File: src/views/Orders.vue (hunk 4 above)
Comment (rathoreprashant) [reply src/views/Orders.vue]: updated

File: src/views/Orders.vue (hunk 5 above)
Comment (rathoreprashant) [reply src/views/Orders.vue]: updated sir

File: src/views/Orders.vue [hunk 6]
```diff
@@ -424,6 +446,25 @@ export default defineComponent({
       }
     });
     this.store.dispatch('webhook/fetchWebhooks')
+
+    const payload = {
```
Comment (adityasharma7) [inline src/views/Orders.vue:450]: Implement a separate method to perform this

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Pull request merged**